module/zfs/dmu_send.c

```diff
@@ -62,21 +63,26 @@
 int zfs_send_corrupt_data = B_FALSE;
 int zfs_send_queue_length = 16 * 1024 * 1024;
 int zfs_recv_queue_length = 16 * 1024 * 1024;
+/* Set this tunable to FALSE to disable setting of DRR_FLAG_FREERECORDS */
+int zfs_send_set_freerecords_bit = B_TRUE;
```

Comment:
Why would you want to set this to false?  You're still going to send the FREE records and FREEOBJECT records, it just won't be marked with the flag.  The flag doesn't prevent the stream from being received on older systems.  Unless the concern is that someone may already have used that flag bit for something else on the receiving system?